application/src/main/kotlin/net/corda/v5/application/flows/RestStartableFlow.kt

```diff
@@ -37,22 +37,22 @@ import net.corda.v5.application.marshalling.JsonMarshallingService
  * - Java:
  *
  * ```java
- * public class MyFlow implements RPCStartableFlow {
+ * public class MyFlow implements RestStartableFlow {
  *
  *     @CordaInject
  *     public JsonMarshallingService jsonMarshallingService;
  *
  *     @Suspendable
  *     @Override
- *     public String call(RPCRequestData requestBody) {
+ *     public String call(RestRequestBody requestBody) {
  *         MyFlowParameters parameters = requestBody.getRequestBodyAs(jsonMarshallingService, MyFlowParameters.class);
  *         ...
  *         return jsonMarshallingService.format(parameters);
  *     }
  * }
  * ```
  */
-interface RPCStartableFlow : Flow {
+interface RestStartableFlow : Flow {
```

Comment:
To me this could be `HTTPStartableFlow`, and the request body `HTTPRequestBody`, since they are agnostic to the architecture / style of our endpoints.

Wdyt @vkolomeyko @lankydan ?

Reply:
I am flexible although @mnesbit mentioned that in the future we might want to start flow from Kafka messages in which case `ExternallyStartableFlow` or `ClientStartableFlow` would make more sense. 
Also FAO @driessamyn

Reply:
I've updated to ClientStartableFlow